cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1935,11 +1935,11 @@ Slice::Gen::DataDefVisitor::visitExceptionStart(const ExceptionPtr& p)
             H << nl << "/// One-shot constructor to initialize all data members.";
             for (const auto& dataMember : allDataMembers)
             {
```

Comment:
I slightly simplified this.
We were storing the entire `DocComment`, when all we actually used was the overview.
So, now we just store the overview instead of copying the entire comment.